frontend/src/main/pages/ProfilePage.js

```diff
@@ -3,9 +3,24 @@ import { Row, Col } from "react-bootstrap";
 import RoleBadge from "main/components/Profile/RoleBadge";
 import { useCurrentUser } from "main/utils/currentUser";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+// import axios from "axios";
 
 const ProfilePage = () => {
 
+    // Stryker disable all
```

Comment:
Please remove the commented out code

Reply:
This is necessary to remove the testing for Modals as mentioned on Slack. I removed my jank "console.log" strategy as it is not good to have in a "production" product or on the main branch